### ISSUE TITLE
The $pkgfile glob pattern matches other files

### ISSUE THREAD

**Issue by yn-** [joanrieu/pacnew-auto#1]
There is ambiguity in saying $pkgfile-\* (line 17). Running on cups packages, for example, might find a wrong package.

**Comment by joanrieu** [comment]
You're right, it would be better to use a regular expression to filter the list.
